lms/static/scripts/frontend_apps/services/grading.ts

```diff
@@ -6,6 +6,9 @@ export type Student = {
 
   /** API URL for posting outcome results. */
   LISOutcomeServiceUrl: string;
+
+  /** LTI user id for this student. */
+  lmsId: string;
```

Comment:
We don't need this value to send the grading itself as the grading LTI api is based on `lis_result_sourcedid`.

In some LMSes those values are the same but that's not guaranteed.


We add it here to be able to track who is being graded in the event.

Reply:
When the frontend makes an API request to submit a grade it passes an authentication token. Is the backend not able to figure out the LMS user ID from the authentication context?

Reply:
Yes, this is the id of the student being graded, not the teacher doing the grading.

This is probably extra confusing by the soup of names here, I maintained the `lmsId` name we have here but maybe it would be easier to use `studentUserId` everywhere.